src/templates/static/legal/datafeed.jsx

```diff
@@ -2,29 +2,29 @@ import React from 'react';
+        <p>{it.L('The Company’s data feed may at times differ slightly from other data feeds found on the Internet (who may themselves differ from each other). Reasons for differences in feed providers include:')}</p>
 
         <ul>
-            <li>{it.L('For forex: The forex market is an OTC (over the counter) market, where banks and other major financial institutions trade currencies amongst themselves without there being any central clearing house. Accordingly, there is no \'official\' price source for forex quotes. Different data feeds will contain quotes from a different sub-set of international banks. Accordingly prices may differ between providers, depending on which bank(s) they obtain prices from.')}</li>
+            <li>{it.L('For forex: The forex market is an OTC (over the counter) market, where banks and other major financial institutions trade currencies amongst themselves without there being any central clearing house. Accordingly, there is no \'official\' price source for forex quotes. Different data feeds will contain quotes from a different sub-set of international banks. Accordingly, prices may differ among providers, depending on which bank(s) they obtain prices from.')}</li>
             <li>{it.L('The smoothing and filtering algorithms described above.')}</li>
-            <li>{it.L('Market-closing times: Please refer to')} <a href={it.url_for('resources/market_timesws')}>{it.L('Trading Times')}</a> {it.L('page for the exact time of settlement for all contracts. Other websites may adopt a different convention (for example some websites choose 4 o\'clock NY time, or 5 o\'clock London time). As a result, the open, high, low and closing prices displayed on our website may differ to those on other websites, due to the market-closing time convention.')}</li>
-            <li>{it.L('Use of bid/ask prices: when the market is illiquid, the data feed may contain many bid and ask prices, without there actually being any traded price for quite a while. By taking an average of the bid/ask price (i.e. bid + ask, divided by 2), a market quote is generated that reflects the current market, without it actually being a traded price. Our system will generate prices from these bid and ask prices, whereas other websites might not. As a result, our website might display ticks that do not appear in the data feeds of other websites.')}</li>
+            <li>{it.L('Market-closing times: The client is asked to refer to')} <a href={it.url_for('resources/market_timesws')}>{it.L('Trading Times')}</a> {it.L('page for the exact time of settlement for all contracts. Other websites may adopt a different convention (for example, some websites choose 4 o\'clock NY time or 5 o\'clock London time). As a result, the open, high, low and closing prices displayed on the Company’s website may differ from those on other websites, due to the market-closing time convention.')}</li>
```

Comment:
I know this is not related to your changes and was here before but this should be like below

Reply:
would be great if you would change it

Reply:
sorry I mistakely commited that, it would be great if you could fix other one mistake from before like this in this page